### ISSUE TITLE
After install not displaying - Laravel 9

### ISSUE THREAD

**Issue by AiFxApp** [barryvdh/laravel-debugbar#1297]
"laravel/framework": "^9.0", 
"php": "^8.0.2",
"livewire/livewire": "^2.10",
Breeze.

I always and forever have used barryvdh/laravel-debugbar with no issues ever. 

I went the standard route of installing as per GITHUB REPO as i have done on countless projects. 

This time the debug bar does not display. 

I have run every possible *:clear / autodump CMD without success? 

Please advice - Thank you.

![image](https://user-images.githubusercontent.com/89227642/154832928-f0cd8234-d2ca-48a5-973f-779a523e521f.png)

**Comment by hopkins385** [comment]
as per documentation: 

> the Debugbar will be enabled when APP_DEBUG is true.

Is in your .env file APP_DEBUG=true ?

**Comment by RhysLees** [comment]
I got the same issue

**Comment by toby1kenobi** [comment]
I seem to have this too (having used debugbar easily before) - Windows 10, Laravel 9, PHP8.1, APP_DEBUG=true, clearing everything I can but can't get debugbar to appear

**Comment by AiFxApp** [comment]
> as per documentation:
> 
> > the Debugbar will be enabled when APP_DEBUG is true.
> 
> Is in your .env file APP_DEBUG=true ?

Yes, as I have mentioned.. I use this in all my projects with success. 

1st : I install the debug bar, as per repo guide
2nd: .env => APP_DEBUG=true

Please note, this only seems to occur on that specific project ??? Every other install is working.. so o copied the code (painful) into a new framework.. and all is well. 

Thank you for your reply, sorry I didn't reply sooner.

**Comment by Wall0fDeath** [comment]
HI!
I have an Aimeos shop installation ( the standalone version ).
I installed this package and I set the .env variables as it was mentioned.

After restart, browser reopen, refresh, dump autoload I still can't see the debug bar.

I had simple laravel project without the Aimeos. 
It worked with that. ( But that was Laravel 8 not 9 as it is in the Aimeos project )

Any idea?

**Comment by joelconty** [comment]
This is happening to me too. I have been using it for many years. Just cant get it to work on Laravel 9.
Laravel Version ........................................................................................................................... 9.22.1  
PHP Version ................................................................................................................................ 8.1.8  
Composer Version ........................................................................................................................... 2.3.3  
Environment ................................................................................................................................ local  
Debug Mode ............................................................................................................................... ENABLED

**Comment by Wall0fDeath-Mithrandir** [comment]
I used Aimeos with Laravel 9 and it seems the debugbar had a problem with the CSP. It didn't show images and it blocked many inline scripts and the debugbar too.

I changed this default in the copy of base.blade.php:

```
<meta http-equiv="Content-Security-Policy" content="base-uri 'self'; default-src 'self' 'nonce-{{ app( 'aimeos.context
' )->get()->nonce() }}'; {{ config( 'shop.csp.frontend', 'style-src \'unsafe-inline\' \'self\'; img-src \'self\' data: https://aimeos.org; fra
me-src https://www.youtube.com https://player.vimeo.com' ) }}">
```

To this for test:

```
<meta http-equiv="Content-Security-Policy" content="default-src *; script-src 'self' 'unsafe-inline'; style-src 'self'
 'unsafe-inline'; media-src *; img-src 'self' data:">
 
```
 Everything started to work, even the debugbar.
 
 
 **This is not a perfect solution**, just a **hint** to where to start. Everyone must write his own CSP handling. :))
 
 Have a nice day!

**Comment by joelconty** [comment]
> 

For those who might be interested, I discovered what was my issue:
In my  routes/api.php I had a line for deactivating debugbar in my api endpoints, like this:
\Debugbar::disable();
Up to laravel 8 this was fine.. It did not affect anything outside my api routes. But that changed in Laravel 9.  So all i had to do in my routes/api.php was put the debugbar disable line inside  the functions i had there , like this:
Route::get('foo/bar', function(){
    \Debugbar::disable();
    return ApiController::getFoo();
});

**Comment by mgw2007** [comment]
hi 
I had same issue and the solution for my case is create folder **storage/debugbar** and give it permissions 
after I create this folder the toolbar appear 
thanks

**Comment by luckydonald** [comment]
Also note that it may not show up due to `CSP_ENABLED=true`. Setting it to `false` (not recommended) did fix it for me.

**Comment by ciceromatias** [comment]
After install and config the extension, try it:

`php artisan optimize`

**Comment by andrewvaca** [comment]
Hi, I've just experience this same strange behavior - debugbar was not visible on local env after fresh install and after one hour of trying every advice I could find I find out it was blocked by Brave's (browser's) anti-trancking shield. I dunno why it was blocked. I just want to share this to maybe save someone some time in future :)

**Comment by beataburczyk** [comment]
> After install and config the extension, try it:
> 
> `php artisan optimize`

This solution also works for me, for listed below components:
"laravel/framework": "^9.2",
"laravel/octane": "^1.2",
"livewire/livewire": "^2.10",
"barryvdh/laravel-debugbar": "^3.7",

**Comment by gabriuss** [comment]
> After install and config the extension, try it:
> 
> `php artisan optimize`

None of the solutions worked for me, this one did. Thank you.

**Comment by nafiesl** [comment]
> hi I had same issue and the solution for my case is create folder **storage/debugbar** and give it permissions after I create this folder the toolbar appear thanks

I had this issue and this solution works for me, thanks @mgw2007

**Comment by makowskid** [comment]
After recent changes in the Debugbar package you need to set

    `DEBUGBAR_ENABLED=true`

inside your `.env` file
